sklearn/model_selection/_search.py

```diff
@@ -1446,6 +1446,9 @@ class GridSearchCV(BaseSearchCV):
             'params'             : [{'kernel': 'poly', 'degree': 2}, ...],
             }
 
+        For an example of visualization and interpretation of GridSearch results,
+        see :ref:`sphx_glr_auto_examples_model_selection_plot_grid_search_stats.py`.
```

Comment:
I like where this link is, but it could be also added to all the other 3 search cv classes where `cv_results_` are present.